cvat/apps/lambda_manager/views.py

```diff
@@ -209,6 +219,11 @@ def invoke(self, db_task, data):
                 for item in response:
                     item["label"] = mapping.get(item["label"])
                 response = [item for item in response if item["label"]]
+            # TODO: Need to add attributes mapping similar to labels.
+            # Currently attribute is expicitely discarded if it is not decalred as supported in function config.
+            if self.attributes:
+                for item in response:
+                    item['attributes'] = [attr for attr in item.get("attributes", []) if attr['name'] in self.attributes[item['label']]]
```

Comment:
@nmanovic I came back to this code after some time and now I'm in doubt is this check really make sense.  Could you please advise?

Here I'm checking that attribute name returned from nuclio function present in `self.attributes` which is `spec` field from `function.yml` in fact (set on line 106). If attribute was not specified in function config it will be discarded. 
But it confuses me now, maybe better to check that attribute name and value returned from nuclio exist in  corresponding label in `db_task` data? 
Which behaviour will be preferable until attributes mapping doesn't implemented in UI?